source/Node.Extensibility.Tests/TestResults/slewi_SHANNONXPS_2017-10-30_11_57_13.trx

```diff
@@ -1,68 +0,0 @@
-﻿<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Not sure what this change has to do with this PR?

Reply:
It was an extraneous test results file that looks like it was committed by accident a while ago. I can separate that out into a separate PR if it offends enough people by leaving it in here :)